### PR TITLE
[FIX] business_requirement_deliverable_project_task_categ

### PR DESCRIPTION
**Pull request by martintorresvm**
- Fix View need to be inherited to could see the Category

### CODE REVIEW

**Comment by martintorresvm** [comment]
@elicoidal @seb-elico Please have a look

**Review by elicoidal — APPROVED** [review]

File: business_requirement_deliverable_project_task_categ/views/business_requirement_deliverable.xml [hunk 1]
```diff
@@ -15,5 +15,35 @@
...
+        <!-- Requirement Deliverable Line Noedit form view-->
+        <record id="view_business_requirement_deliverable_form" model="ir.ui.view">
+            <field name="name">business.requirement.deliverable.form.noedit</field>
+            <field name="model">business.requirement.deliverable</field>
+            <field name="inherit_id"
+                   ref="business_requirement_deliverable.view_business_requirement_deliverable_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='resource_ids']/tree[@editable='1']/field[@name='name']"
+                       position="after">
+                       <field name="categ_id"/>
```
Comment (seb-elico) [inline business_requirement_deliverable_project_task_categ/views/business_requirement_deliverable.xml]: Could you please fix the indentation of this line?

**Review by seb-elico — CHANGES_REQUESTED** [review]
Almost LGTM

**Comment by martintorresvm** [comment]
@seb-elico Please approve if ok

**Review by seb-elico — APPROVED** [review]
@elicoidal @victormartinelicocorp LGTM

File: business_requirement_deliverable_project_task_categ/views/business_requirement_deliverable.xml [hunk 2]
```diff
@@ -12,6 +12,36 @@
                     <field name="categ_id" 
                            attrs="{'readonly':[('resource_type','=','procurement')],'required':[('resource_type','=','task')]}"/>
                 </field>
+            </field>business_requirement_deliverable_project_task_categ
```
Comment (dreispt) [inline business_requirement_deliverable_project_task_categ/views/business_requirement_deliverable.xml]: This line doesn't look right...

File: business_requirement_deliverable_project_task_categ/views/business_requirement_deliverable.xml [hunk 3]
```diff
@@ -12,6 +12,36 @@
...
+            <field name="name">business.requirement.resource.search</field>
+            <field name="model">business.requirement.resource</field>
+            <field name="inherit_id"
+                   ref="business_requirement_deliverable.view_business_requirement_resource_search"/>
+            <field name="arch" type="xml">
+                <field name="product_id" position="after">
+                    <field name="categ_id"/>
+                </field>
+                <group expand="0" position="inside">
+                    <filter string="Category" icon="terp-go-home" context="{'group_by':'categ_id'}"/>
```
Comment (dreispt) [inline business_requirement_deliverable_project_task_categ/views/business_requirement_deliverable.xml]: Icons are deprecated, afaict.

File: business_requirement_deliverable_project_task_categ/views/business_requirement_deliverable.xml [hunk 4]
```diff
@@ -12,6 +12,36 @@
...
+        </record>
+
+        <!-- Requirement Deliverable Line Noedit form view-->
+        <record id="view_business_requirement_deliverable_form" model="ir.ui.view">
+            <field name="name">business.requirement.deliverable.form.noedit</field>
+            <field name="model">business.requirement.deliverable</field>
+            <field name="inherit_id"
+                   ref="business_requirement_deliverable.view_business_requirement_deliverable_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='resource_ids']/tree[@editable='1']/field[@name='name']"
```
Comment (dreispt) [inline business_requirement_deliverable_project_task_categ/views/business_requirement_deliverable.xml]: Is the `[@editable='1']` really needed? Can we get away with a more flexible query, such as `tree[0]`?

**Comment by elicoidal** [comment]
@victormartinelicocorp please give some love to Travis?

**Comment by martintorresvm** [comment]
@elicoidal Done.

Also I fixed @dreispt comments.

**Review by dreispt — APPROVED** [review]

**Comment by elicoidal** [comment]
Tahnks @dreispt !

**Pull request merged**